AstroAdapt.GraphQL/Program.cs

```diff
@@ -16,14 +17,17 @@
 builder.Services.AddSingleton<SolutionProcessingService>();
 
 builder.Services.AddGraphQLServer()
-    .AddProjections()
-    .AddFiltering()
-    .AddSorting()
     .AddQueryType<Query>()
-    .AddType<SolutionInputType>()
     .AddMutationType<Mutation>()
-    .AddTypeExtension<ComponentExtensions>()
     .AddSubscriptionType<Subscription>()
+    .AddAstroTypes()
```

Comment:
Here I am adding the `AddAstroTypes`. All type extensions, types, and dataloader in this assembly are collected at build time and we generate for those this extension method. Hence you no longer have to keep track of things that you need to register. Plus there is no perf hit like with assembly scanning since we do it on build time.